cluster-up/cluster/kind-k8s-1.17/provider.sh

```diff
@@ -39,10 +46,7 @@ function up() {
     # Since Kind provider uses containers as nodes, the UUID on all of them will be the same,
     # and Migration by libvirt would be blocked, because migrate between the same UUID is forbidden.
     # Enable PodPreset so we can use it in order to mount a fake UUID for each launcher pod.
-    $DOCKER exec kind-1.17-control-plane bash -c 'sed -i \
-    -e "s/NodeRestriction/NodeRestriction,PodPreset/" \
-    -e "/NodeRestriction,PodPreset/ a\    - --runtime-config=settings.k8s.io/v1alpha1=true" \
-    /etc/kubernetes/manifests/kube-apiserver.yaml'
+    podpreset::expose_unique_product_uuid_per_node "$CLUSTER_NAME" "$TESTS_NS"
```

Comment:
We enable PodPreset on SRIOV and GPU providers to enable migration on our test suite
since this provider is not used on CI we dont need to enable PodPreset.
I think you can remove it.

Reply:
Yes, I know, I am doing a migration e2e test on ARM via a kind provider (I only have one ARM server, and ARM server does not support nested virtualization, I can only do e2e tests on the kind provider.)
It would be nice if we can keep the PodPreset in the script, but I can remove the line if you think it is not appropriate.

By the way, I am also doing CDI e2e test on the kind provider, it seems that CDI can only work appropriately on k8s 1.19 and above, can I upgrade the kind-k8s-1.17 to kind-k8s-1.19?

Reply:
sure, just node that best that kubevirt doesn't install CDI for kind providers